deploy/gke/redaction/README.md

```diff
@@ -68,4 +68,105 @@ When you are done, you can clean up everything you've done with the following st
 
 ```
 kubectl delete namespace $OTEL_NAMESPACE
-```
\ No newline at end of file
+```
+
+## Try it out yourself (Optional)
+
+This section contains steps that you can follow to actually see redaction you setup earlier in action. Since there is no expected outcome from our kubernetes deployment due to a lack of telemetry data, the redaction processor setup in our collector cofiguration is not applied to anything. In this section we will introduce a source of telemetry data to actually see the redaction in action.  
+
+### Prerequisites 
+You should be able to follow all steps mentioned in this README till [Verify the Deployment](#verify-the-deployment) successfully. This ensures that there are no permissions issues and you are able to succeesfully run a GKE cluster and connect to it. 
+
+### Source of telemetry data
+
+We would be using JSON files containing telemetry data in OTLP format as the source of telemetry data. A sample file has been provided here - [testdata.json](./otlp-data/testdata.json). The collector running in the cluster would read this file and treat the data in this file as if it were coming from a running application. 
+
+### Updating the collector configuration to receive data from JSON file
+
+Next, we would update our collector configuration file - [otel-config.yaml](./otel-config.yaml) to add a reciever that is able to receive telemetry data from the added JSON file. 
+
+Under the `receivers` section in the config file, add the following configuration - 
```

Comment:
This step might be error prone for users not familiar to the otel-collector's config structure.  It also doens't have much educational value, IMO (user just copy-pastes).

Can we either:
* Incorporate this into the original otel-config.yaml file, or
* Make a separate otel-config-example.yaml file?

Reply:
@damemi mentioned that a more comprehensive example code here would be beneficial for the users - since there are multiple receiver sections in the otel config. 

But I like the idea of providing a `otel-config-redaction.yaml` which contains the final expected yaml configuration. We can then also add a few comments within the file and link that file in README for reference.